### ISSUE TITLE
Blender 4.0.2 fails to import the add-on version 4.0.4.2 correctly

### ISSUE THREAD

**Issue by BLANKE1** [teamneoneko/Cats-Blender-Plugin#59]
Upon enabling the add-on in blender, the error in the code block occurs. The previous version (4.0.4.1) works fine.
```
Traceback (most recent call last):
  File "C:\Users\BLANKE\Apps\Blender\blender-4.0.2-windows-x64\4.0\scripts\modules\addon_utils.py", line 364, in enable
    mod = importlib.import_module(module_name)
  File "C:\Users\BLANKE\Apps\Blender\blender-4.0.2-windows-x64\4.0\python\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\BLANKE\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\Cats-Blender-Plugin-Unofficial--blender-40\__init__.py", line 47, in <module>
    from . import updater
  File "C:\Users\BLANKE\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\Cats-Blender-Plugin-Unofficial--blender-40\updater.py", line 17, in <module>
    from .tools.common import wrap_dynamic_enum_items
  File "C:\Users\BLANKE\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\Cats-Blender-Plugin-Unofficial--blender-40\tools\__init__.py", line 7, in <module>
    from . import armature
  File "C:\Users\BLANKE\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\Cats-Blender-Plugin-Unofficial--blender-40\tools\armature.py", line 8, in <module>
    from . import common as Common
  File "C:\Users\BLANKE\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\Cats-Blender-Plugin-Unofficial--blender-40\tools\common.py", line 31, in <module>
    from . import iconloader as Iconloader
  File "C:\Users\BLANKE\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\Cats-Blender-Plugin-Unofficial--blender-40\tools\iconloader.py", line 11, in <module>
    from . import settings as Settings
  File "C:\Users\BLANKE\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\Cats-Blender-Plugin-Unofficial--blender-40\tools\settings.py", line 18, in <module>
    from . import translate as Translate
  File "C:\Users\BLANKE\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\Cats-Blender-Plugin-Unofficial--blender-40\tools\translate.py", line 37, in <module>
    cats_dir.mkdir(exist_ok=True)
  File "C:\Users\BLANKE\Apps\Blender\blender-4.0.2-windows-x64\4.0\python\lib\pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\BLANKE\\Documents\\cats'
```

**Comment by BLANKE1** [comment]
This seems to stem from the script not properly finding out the windows documents folder if it's not in the default C drive location. My current documents folder is located at D:\Documents

**Comment by RinaDev-xyz** [comment]
Are you using the windows store version of cats?

**Comment by BLANKE1** [comment]
I'm using a portable zip version of blender stored in my C drive, specifically at C:\Users\BLANKE\Apps\Blender\blender-4.0.2-windows-x64.

**Comment by RinaDev-xyz** [comment]
Let me test a portable version as I can't reproduce this using an installed version.

**Comment by RinaDev-xyz** [comment]
Yea I can't seem to re-produce this issue, can you try the following. Also what version of windows are you using?

1: Check your documents to see if there is a folder called cats, if there is remove it.
2: Remove cats from blender.
3: Close Blender.
4: Open blender and install cats again.
5: Restart Blender.

If the issue still persists I will need to look into further.

**Comment by BLANKE1** [comment]
This is on Windows 11 version 23H2 build 22631.3155

Error still occurs after doing all these steps, it's looking for a documents folder in my C: drive which in Windows it's been moved to my D: drive natively through doing it in the properties.

**Comment by RinaDev-xyz** [comment]
Ahhhh, ok this is why you are having issues then, was going to ask this.

Cats is trying to make this folder, this is for a new feature which I introduced in this version of cats, however it presumes you don't have any weird variables like this.

for now use the previous version of cats 4.0.4.1 won't do this, I going to try and release a new version which hopefully takes this variable into account later today.

https://github.com/Yusarina/Cats-Blender-Plugin-Unofficial-/releases/tag/4.0.4.1 should work just fine.

Also was asking windows version as some user was having similar issues on Windows 10, but your issue is unrelated to that.

**Comment by RinaDev-xyz** [comment]
This issue should now be fixed on the dev branch for blender versions 3.6, 4.0 and 4.1.

The addon should just throw an error at the bottom asking you to set your own folder and shouldn't crash.
You can now set your own path in settings and updates of the cats UI.

Please test the dev version of your blender and let me know if there is any further issues.

Thanks!

**Comment by BLANKE1** [comment]
The addon no longer breaks on load when using the development branch.

**Comment by RinaDev-xyz** [comment]
Perfect, I will push it to release most likely tomorrow then.

**Comment by RinaDev-xyz** [comment]
This has been released to version 3.6.6.4 for blender 3.6 and 4.0.4.3 for blender 4.0.

Issue has now been closed, if you have further issues relating to this issue feel free to re-open.

Thanks!
